README.md

```diff
@@ -30,6 +44,8 @@ Please run the server and see [http://localhost:8000/entries](http://localhost:8
 
 ### `GET /api/pulse/nonce/`
 
+**Note:** This API route does not have a version in it since it does not rely on data in the database.
```

Comment:
For ease of development, would it be possible to have all routes accepted the `vX` infix, so that on the client side the API prefix can be kept a single string? (e.g. Set `/api/pulse/v1/` as API root, without having to main two different roots for versioned vs. unversioned?)